modules/tinymce/src/core/main/ts/api/EditorCommands.ts

```diff
@@ -52,81 +39,49 @@ export interface EditorCommandsConstructor {
 
 class EditorCommands {
   private readonly editor: Editor;
-  private selectionBookmark: Bookmark;
   private commands: Commands = { state: {}, exec: {}, value: {}};
 
   public constructor(editor: Editor) {
     this.editor = editor;
-
-    this.setupCommands(editor);
   }
 
   /**
-   * Executes the specified command.
+   * Executes a registered command on the current instance. A list of available commands can be found in
+   * the <a href="https://www.tiny.cloud/docs/advanced/editor-command-identifiers/">command identifiers</a> documentation.
    *
    * @method execCommand
-   * @param {String} command Command to execute.
-   * @param {Boolean} ui Optional user interface state.
-   * @param {Object} value Optional value for command.
-   * @param {Object} args Optional extra arguments to the execCommand.
-   * @return {Boolean} true/false if the command was found or not.
+   * @param {String} cmd Command name to execute, for example mceLink or Bold.
+   * @param {Boolean} ui True/false state if a UI (dialog) should be presented or not.
+   * @param {mixed} value Optional command value, this can be anything.
```

Comment:
`mixed` isn't a valid type... I'd suggest using something like what's used in `EditorManager` instead or just saying `any`.
```suggestion
   * @param {Object/Array/String/Number/Boolean} value Optional command value, this can be anything.
```